rules/telemetry.mk

```diff
@@ -0,0 +1,5 @@
+# telemetry package
+
+TELEMETRY = telemetry_0.1_amd64.deb
```

Comment:
can you rename the package name to sonic-telemetry?

Reply:
Ok.